packages/react-code-editor/package.json

```diff
@@ -38,10 +38,11 @@
   "peerDependencies": {
     "react": "^17 || ^18",
     "react-dom": "^17 || ^18",
-    "react-monaco-editor": "^0.51.0"
```

Comment:
I believe changing the Peer dependency here is breaking.  We would want to introduce this as next.